pymilo/utils/test_pymilo.py

```diff
@@ -25,9 +27,10 @@ def pymilo_test(model, model_name, test_data):
     """
     x_test, _ = test_data
 
+    export_model_path = "exported_linear_models" if is_linear_model(model) else "exported_neural_networks" if is_neural_network(model) else None
```

Comment:
Is there any more scalable approach (like dictionaries)?

Reply:
Sure, you're right.